### DISCUSSION TITLE
Thanks, for your attention of this discussion.

### DISCUSSION THREAD

**Discussion by happysmile0227** [PurpleMonkey729/GitHub-Profile-Achievement-Badge#4] [General]
Thanks, for your attention of this discussion.
If you need the quick draw badges, please conform the following link.
https://github.com/orgs/community/discussions/20645
Best regards!

_Originally posted by @PurpleMonkey729 in https://github.com/PurpleMonkey729/GitHub-Profile-Achievement-Badge/discussions/1#discussioncomment-6332277_

**Comment by PurpleMonkey729** [comment]
Well, I think you get the constant reply. Thanks.
I will close this issue.

**Comment by PurpleMonkey729** [comment]
Closed
